tests/test_auto_parallel/test_tensor_shard/test_device/test_alpha_beta.py

```diff
@@ -0,0 +1,4 @@
+from colossalai.auto_parallel.tensor_shard.utils import get_alpha_beta
+
+def test_get_alpha_beta():
+  get_alpha_beta(2,1)
```

Comment:
Is there any assertion statements which can be added to verify the correctness of the profiling results.

Reply:
Meanwhile, it seems that you didn't use pre-commit hooks to format your code, please refer to the `Contributing.md` for details.

Reply:
It doesn't need assertion because it is a profiling tool. The output is unknown before running the program.